src/shared/components/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -125,9 +125,9 @@ export class CancerSummaryContent extends React.Component<ICancerSummaryContentP
                     });
                     return memo;
                 }, [] as IBarGraphDataset[]);
+                const orderedAlts = ['multiple', 'amp', 'mutated', 'fusion', 'gain', 'homdel', 'hetloss',
```

Comment:
it would be best to declare this somewhere else so its not rebuilt on every call. Also, it would be great to somehow link this order with the order below to DRY it up

Reply:
+1. The order should be defined once.

Reply:
... and amp is still not in the correct position.